contracts/deploy/deployActions.js

```diff
@@ -1019,23 +1021,27 @@ const deployOracles = async () => {
   let args = [];
   if (isMainnet) {
     oracleContract = "OracleRouter";
-  } else if (isHolesky) {
+  } else if (isHoleskyOrFork) {
     oracleContract = "OETHFixedOracle";
     contractName = "OETHOracleRouter";
     args = [addresses.zero];
+  } else if (isSonicOrFork) {
+    oracleContract = "OETHFixedOracle";
+    contractName = "SOOracleRouter";
```

Comment:
I'll rename this to OSonicOracleRouter to be consistent with the other Sonic contract names

Reply:
thanks 👍